src/Microsoft.Health.Dicom.Blob/Features/Export/ExportPatternPlaceholder.cs

```diff
@@ -0,0 +1,17 @@
+﻿// -------------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License (MIT). See LICENSE in the repo root for license information.
+// -------------------------------------------------------------------------------------------------
+
+using System;
+
+[Flags]
+internal enum ExportPatternPlaceholders : sbyte
```

Comment:
Do not understand. Do we need this? Should we need to parameterize this?

Reply:
As discussed offline, I went all in on the path specification lol. I have made this `internal` for now